HTML/Rebooting M-OS.html

```diff
@@ -455,7 +455,7 @@ <h2>Final modifications</h2>
                         </tr>
                         <tr>
                             <td>3</td>
-                            <td>2A - (C % B)</td>
+                            <td>2A - (B % C)</td>
```

Comment:
Please clarify with author again about all the changes on this file. Thank you.

Reply:
I confirmed with the author the current manual i have in my pr is correct.